Vagrantfile

```diff
@@ -37,7 +37,7 @@ fail 'Non-unique hostnames' if @network.collect { |_, v| v[:hostname] }.uniq!
 fail 'Non-unique ports' if @network.collect { |_, v| v[:ports].keys }.flat_map { |v| v }.uniq!
 
 def default_omnibus(config)
-  config.omnibus.chef_version = :latest
+  config.omnibus.chef_version = '12.17.44'
```

Comment:
Looks like we are locking chef to 12.4.3 only for ruby 2.1.6. I didn't run into any issues with the 2.3.1 build which is why I didn't lock the version in the Gemfile.
https://travis-ci.org/cerner/cerner_splunk/builds/202454618

Reason why I locked it here is explained in this [comment](https://github.com/cerner/cerner_splunk/pull/140#discussion_r101807354).